bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -93,17 +93,28 @@ export async function getDependentStoryFiles(
   const namesById: Record<number, string> = {};
   const reasonsById: Record<number, string[]> = {};
   const csfGlobsByName: Record<string, true> = {};
-
+  const CSF_REGEX = /.\/src.*?sync/g;
+  const URL_PARAM_REGEX = /(\?.*)|(#.*)/g;
```

Comment:
I'm not sure that webpack "urls" can have hashes in them so I'd be inclined to keep it simple and leave hashes out.